src/flows/context/flow.query.tsx

```diff
@@ -242,39 +245,27 @@ export const FlowQueryProvider: FC = ({children}) => {
     Promise.all(
       map
         .reduce((acc, curr) => {
-          acc.push({
-            text: curr.text,
-            instances: curr.instances.filter(i => !i.modifier).map(i => i.id),
-          })
-
           return acc.concat(
             curr.instances
               .filter(i => i.modifier)
               .map(i => ({
                 text: i.modifier,
-                instances: [i.id],
+                instance: i.id,
               }))
           )
         }, [])
-        .filter(stage => !!stage.instances.length)
```

Comment:
what was this doing and what's changed to render this step redundant?

Reply:
we dont do the squishing anymore. first.. by conventions (which caused this bug), now codified